drivers/gles3/storage/texture_storage.cpp

```diff
@@ -1475,7 +1475,7 @@ void TextureStorage::update_texture_atlas() {
 		//generate atlas
 		Vector<TextureAtlas::SortItem> itemsv;
 		itemsv.resize(texture_atlas.textures.size());
-		int base_size = 8;
+		unsigned int base_size = 8;
```

Comment:
We use `uint32_t` instead of `unsigned int`
```suggestion
		uint32_t base_size = 8;
```

Reply:
thanks! fixed